markdown_lab/core/config.py

```
@@ -0,0 +1,200 @@
+
+        if config_path.suffix.lower() == '.json':
+            import json
+            with open(config_path, 'r') as f:
+                config_dict = json.load(f)
+        elif config_path.suffix.lower() in ['.yml', '.yaml']:
+            try:
+                import yaml
+                with open(config_path, 'r') as f:
+                    config_dict = yaml.safe_load(f)
+            except ImportError:
+                raise ImportError("PyYAML is required to load YAML configuration files")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)